params/two_inversions.json

```diff
@@ -2,30 +2,33 @@
     "id": "two_inversions",
     "name": "Two Sphere Inversions",
     "algorithm": "chaos_sets",
-    "initial_set": {
-        "type": "box",
-        "dims": [0.01, 1.0, 8.0],
+    "cluster": {
+        "type": "quad",
         "center": [0, 0, 0],
+        "dims": [1, 8],
+        "x_dir": [0, 1, 0],
+        "y_dir": [0, 0, 1],
         "color": [0.3, 0.8, 0.4],
-        "num_points": 100
+        "num_points": 1000
     },
     "plotter": {
-        "node_capacity": 1000,
         "radius": 8
     },
-    "initial_set_copies": 10,
+    "cluster_copies": 3,
     "iters": 100,
     "ifs": {
         "chooser": "uniform",
         "xforms": [
             ["chain", [
               ["translate", -1, 0, 0],
               ["invert"],
+              ["rotate", 1, 0, 0, 90],
```

Comment:
I kinda like it better without the rotation, though it was a cool experiment. remove the `rotate`